src/core/Akka.Persistence/Persistent.cs

```diff
@@ -369,7 +369,7 @@ public IPersistentRepresentation WithManifest(string manifest)
         /// <inheritdoc />
         public IPersistentRepresentation Update(long sequenceNr, string persistenceId, bool isDeleted, IActorRef sender, string writerGuid)
         {
-            return new Persistent(payload: Payload, sequenceNr: sequenceNr, persistenceId: persistenceId, manifest: Manifest, isDeleted: isDeleted, sender: sender, writerGuid: writerGuid);
+            return new Persistent(payload: Payload, sequenceNr: sequenceNr, persistenceId: persistenceId, manifest: Manifest, isDeleted: isDeleted, sender: sender, writerGuid: writerGuid, Timestamp);
```

Comment:
I didn't change this because it hasn't in the JVM and no test seems broken

Reply:
I'll double check that

Reply:
Reason I added it - seems like we could set the timestamp and then forget it with this method.

Reply:
Fixed